wiki/Ranking_Criteria/Skinning/en.md

```diff
@@ -0,0 +1,331 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
```

Comment:
add a period `.` to the end of this sentence (which ironically this one doesn't \:p )